src/rhsmlib/facts/cloud_facts.py

```diff
@@ -134,6 +135,8 @@ def get_azure_facts(self) -> Dict[str, str]:
                     facts["azure_sku"] = values["compute"]["sku"]
                 if "offer" in values["compute"]:
                     facts["azure_offer"] = values["compute"]["offer"]
+                if "offer" in values["compute"]:
+                    facts["azure_subscription_id"] = values["compute"]["subscriptionId"]
```

Comment:
shouldn't the `if` have `if "subscriptionId" in values["compute"]` instead of `"offer"`?

Reply:
Good catch ... copy/paste error 🤦 . Fix added.